data/anvil_uses.json

```diff
@@ -0,0 +1,3 @@
+{
+    "regular": [0, 1, 3, 7, 15, 31]
```

Comment:
The formula for this data is n = n*2+1 no need to store it

Reply:
it is for the first bit but (2 * 2 + 1 !== 7) & (3 * 2 + 1 != 15)

Reply:
no, its a recurrence relation:
* 0*2+1 = 1
* 1*2+1 = 3
* 3*2+1 = 7
* 7*2+1 = 15
* 15*2+1 = 31

Reply:
add converting as static functions with tests

Reply:
done